sunpy/coordinates/utils.py

```diff
@@ -265,3 +265,72 @@ def coordinates(self, points=None):
                         obstime=self.obstime,
                         observer=self.observer,
                         frame=Heliocentric).transform_to(self.start_frame)
+
+
+def get_rectangle_coordinates(bottom_left, *, top_right = None, width: u.deg = None, height: u.deg = None):
+    """
+    Specify a rectangular region of interest in longitude and latitude in a given coordinate frame.
+
+    Parameters
+    ----------
+    bottom_left : `~astropy.coordinates.BaseCoordinateFrame` or `~astropy.coordinates.SkyCoord`
+        The bottom-left coordinate of the rectangle.
+        Supports passing both ``bottom_left`` and ``top_right`` as an array of shape ``(2,)``.
+    top_right : `~astropy.coordinates.BaseCoordinateFrame` or `~astropy.coordinates.SkyCoord`
+        The top-right coordinate of the rectangle. Can only be omitted if
+        ``bottom_left`` has shape ``(2,)``.
```

Comment:
You can also omit it if you pass `width` and `height`.
```suggestion
        The top-right coordinate of the rectangle.
```